cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1097,16 +1095,8 @@ public void getServiceInstances(HttpRequest request, HttpResponder responder,
     getServiceInstances(request, responder, namespaceId, appId, serviceId, serviceId);
   }
 
-  /**
-   * Return the number of instances for the given runnable of a service.
-   */
-  @GET
-  @Path("/apps/{app-id}/services/{service-id}/runnables/{runnable-name}/instances")
-  public void getServiceInstances(HttpRequest request, HttpResponder responder,
-                                  @PathParam("namespace-id") String namespaceId,
-                                  @PathParam("app-id") String appId,
-                                  @PathParam("service-id") String serviceId,
-                                  @PathParam("runnable-name") String runnableName) {
+  void getServiceInstances(HttpRequest request, HttpResponder responder,
```

Comment:
Oh ok - so the service handler is sticking around in v3, but simply the workers are being removed.
And so this method is still used for the handler runnable.